src/data/ff-build.yaml

```diff
@@ -1,9 +1,7 @@
-name: Firefox - Development Environment
-summary: The harnesses, code analysis, and linting tools surrounding the building of Firefox.
+name: Firefox - Development Build System Environment
```

Comment:
That seems like a lot of general words.  Is this a team name, or is there something more succinct or specific?

Reply:
Yeah... I kinda want to make it after devtools & co, but I could drop the `Environment` part.